Sources/swift-parser-cli/Commands/PerformanceTest.swift

```diff
@@ -37,12 +41,40 @@ struct PerformanceTest: ParsableCommand {
       .filter { $0.pathExtension == "swift" }
       .map { try Data(contentsOf: $0) }
 
+    var fileTransition: [Data: (SourceFileSyntax, IncrementalParseNodeAffectRangeCollector)] = [:]
+
+    /// The initial parse
+    for file in files {
+      let incrementalParseAffectRangeCollector = self.incrementalParse ? IncrementalParseNodeAffectRangeCollector() : nil
+      file.withUnsafeBytes { buf in
+        let tree = Parser.parse(
+          source: buf.bindMemory(to: UInt8.self),
+          parseNodeAffectRange: incrementalParseAffectRangeCollector
+        )
+
+        if self.incrementalParse == true {
+          fileTransition[file] = (tree, incrementalParseAffectRangeCollector!)
+        }
+      }
+    }
+
```

Comment:
We only need to do the initial parse if `self.incrementalParse` is `true`, right? You could just put the entire loop in a `if self.incrementalParse` and then you don’t need to check that variable inside the loop.